release.md

```diff
@@ -12,15 +12,46 @@ latest commit to master can be installed with `pip install --pre cirq`.
 
 We follow [semantic versioning](https://semver.org/) for labeling our
 releases.  Versions are labeled MAJOR.MINOR.PATCH where each of these
-is a numerical value. The rules for versions changes are:
-* Before MAJOR becomes 1, updates to MINOR can and will make changes to
-public facing apis and interfaces.
-* After MAJOR becomes 1, updates that break the public facing api
-or interface need to update MAJOR version.
-* MINOR updates have to be backwards compatible (after MAJOR>=1).
-* PATCH updates are for bug fixes.
-
-Versions based on unreleased branches of master will be suffixed with ".dev".
+is a numerical value. The following guarantees are provided:
+
+1. All packages released at the same time from the Cirq repository will share
+   the same [Semantic Versioning 2.0.0](https://semver.org/) number.
+   1. Packages may have different version numbers in the future, at which point
+      this policy will be updated.
+2. Libraries in the `cirq-core` directory (with the exception of
+   `cirq-core/cirq/contrib`) adhere to the guarantees outlined in the Semantic
+   Versioning specification. In summary: “Bug fixes not affecting the API
+   increment the patch version, backwards compatible API additions/changes
+   increment the minor version, and backwards incompatible API changes increment
+   the major version.”
+3. The contrib directory (at `cirq-core/cirq/contrib`) follows Semantic
+   Versioning except the MINOR version increment policy: releases with MINOR
+   version increments may contain backward-incompatible functionality changes to
+   its public API.
+   1. May be changed to strictly follow Semantic Versioning in the future, at
+      which point this policy will be updated.
+4. Cirq vendor directories (`cirq-aqt`, `cirq-google`, `cirq-ionq`, etc.) follow
+   Semantic Versioning except the MINOR version increment policy: each vendor
+   directory has a separate policy on whether MINOR version increments provide
+   backward-compatibility guarantees, as described in `version_policy.md` in the
+   respective directory.
+   1. If `version_policy.md` does not exist in a particular vendor directory,
+      MINOR version increments may contain backward-incompatible functionality
+      changes to its public API.
+   2. For each vendor directory, version policies may be modified to strictly
+      follow Semantic Versioning in the future.
```

Comment:
So, if I understand correctly, the version of the vendor packages will still track the version of `cirq-core`, right?

Reply:
Thanks for your review! Yes that's right.